hack/run-functests.sh

```diff
@@ -34,6 +32,9 @@ elif [ "$FEATURES" == "" ]; then
   exit 1
 else
   FOCUS="-ginkgo.focus="$(echo "$FEATURES" | tr ' ' '|')
```

Comment:
In a earlier comment above you wrote:
> i introduced a new var to focus on tests (FOCUS_TESTS), so FEATURES can be just features names.

Should this line be removed?

Reply:
removed why? we focus on features (with FEATURES) or on tests (with FOCUS_TESTS)

Reply:
can you please extend a bit more the doc to explain the difference between the FEATURES and the FOCUS_TESTS.

correct me if I am wrong here but if I understand this right if I want to focus on a specific sriov test I will use both configs right?

```
FEATURES=sriov FOCUS_TESTS="sriov resource injector...."
```

Reply:
right